source/Nevermore/Advanced/Queryable/AsyncEnumerableAdapter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Nevermore.Advanced.Queryable
+{
+    internal static class AsyncEnumerableAdapter
+    {
+        public static async Task<IEnumerable> ConvertToEnumerable(object asyncEnumerable, Type sequenceType, CancellationToken cancellationToken)
+        {
+            var asyncEnumerableType = typeof(IAsyncEnumerable<>).MakeGenericType(sequenceType);
+            var asyncEnumeratorType = typeof(IAsyncEnumerator<>).MakeGenericType(sequenceType);
+
+            var getAsyncEnumeratorMethod = asyncEnumerableType.GetRuntimeMethod("GetAsyncEnumerator", new[] { typeof(CancellationToken) });
```

Comment:
🤔  I hadn't seen `GetRuntimeMethod` used before and it doesn't appear anywhere in the Octopus.Server codebase.

Decompiler shows it's just a wrapper which calls `type.GetMethod`, but the Microsoft Docs are useless and don't offer any explanation as to why there are two methods which do the same thing.

It does appear to be used in other places in Nevermore so keep-calm-carry on I guess?